test/Microsoft.SqlTools.ServiceLayer.IntegrationTests/Connection/ReliableConnectionTests.cs

```diff
@@ -356,11 +373,11 @@ public void TestIsDatabaseReadonlyWithNullBuilder()
         /// Verify ANSI_NULL and QUOTED_IDENTIFIER settings can be set and retrieved for a session
         /// </summary>
         [Fact]
-        public void VerifyAnsiNullAndQuotedIdentifierSettingsReplayed()
+        public async void VerifyAnsiNullAndQuotedIdentifierSettingsReplayed()
         {
-            TestUtils.RunIfWindows(() =>
+            await TestUtils.RunIfWindowsAsync(async () =>
```

Comment:
`RunIfWindowsAsync` doesn't make much sense anymore - the only test code that needs this are the Windows-specific Credentials tests, or Integrated-auth specific tests (which wouldn't many).